### DISCUSSION TITLE
Is there any plan to support chrome?

### DISCUSSION THREAD

**Discussion by dhruv0000** [piroor/treestyletab#3430] [Q&A] [answered]
Is there any plan to support chrome? Is there a technical reason we can't? I am not aware about how open chrome apis (or any other interfaces) are then firefox apis, is that the reason? 
Please feel free to point me to a documentation/blog that explains if what I have asked here is very trivial 😛, would love to learn more.

**Comment by dhruv0000** [comment]
I know that dark reader was able to support firefox too. https://github.com/darkreader/darkreader. This makes me believe chrome has less API(or whatever interfaces it's called) less exposed than firefox. Am I right?

**Reply by dhruv0000** [reply]
@piroor

**Reply by irvinm** [reply, marked as answer]
You may want to read over this:  https://github.com/piroor/treestyletab/issues/3354
